app/Jobs/ApplicationDeploymentJob.php

```
@@ -2447,14 +2631,108 @@ private function generate_build_env_variables()
+            $secret_file = "{$this->secrets_dir}/{$key}";
+            $escaped_value = base64_encode($value);
+
+            $this->execute_remote_command([executeInDocker($this->deployment_uuid,
+                "echo '{$escaped_value}' | base64 -d > {$secret_file} && chmod 600 {$secret_file}"
+            ), 'hidden' => true]);
+
+            $this->build_secrets->push("--secret id={$key},src={$secret_file}");
+        }
+
+        $this->build_secrets = $this->build_secrets->implode(' ');
+    }
```

PHPMD (2.15.0)

The method generate_build_secrets is not named in camelCase. (undefined)

(CamelCaseMethodName)